server/server.ts

```diff
@@ -86,6 +86,7 @@ export default async function (
 	const app = express();
 
 	if (options.dev) {
+		// @ts-ignore
```

Comment:
Same thing here, needs a comment as to why it's safe to ignore or a fix of the root cause